### ISSUE TITLE
non-2020-09-14 methods have been removed in version 8, should not be removed until version 9

### ISSUE THREAD

**Issue by phoenixy1** [plaid/plaid-node#377]
Related to undoing https://github.com/plaid/plaid-node/pull/361

Because we did not version pin this release, we need to retain those methods

**Comment by otherchen** [comment]
Good catch. We're actually planning on releasing an 8.1.0 to add back the endpoints that we removed.

**Comment by otherchen** [comment]
This has been addressed in https://github.com/plaid/plaid-node/blob/master/CHANGELOG.md#810
